packages/terra-button/tests/wdio/button-spec.js

```diff
@@ -153,8 +153,12 @@ describe('Button', () => {
     Terra.should.beAccessible({ context: '#action' });
```

Comment:
I'm a little curious, there are quite a few screenshot changes here than increased the image size but actually shows now diff for the screenshot for non-theming screenshots. Do we need to update these?

Reply:
I have resolved the screenshot size issue. https://github.com/cerner/terra-core/pull/2358/commits/df5f92ded817f7bc3b7ce2cf1fb358ea5c77b11b

I am curious as to why the screenshot size changes in the 'reference' directory when I tried to regenerate all the reference screenshots for `terra-button` for instance without making any code changes on 'master' branch.